packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -79,27 +79,28 @@ const Redirect = ({
             if (redirect_to) {
                 let pathname = '';
                 let hash = '';
+                const wallet_route = is_next_wallet ? routes.wallets : routes.traders_hub;
```

Comment:
naming: target_route ? 
its not always wallet_route, kind of scary to see that redirects are always to wallets (they are not, but name suggests that redirect is always to wallet)